x-pack/plugins/enterprise_search/public/applications/app_search/components/engine_creation/engine_creation.test.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import '../../../__mocks__/shallow_useeffect.mock';
+import { setMockValues } from '../../../__mocks__';
+
+import { EngineCreation } from './';
+
+describe('EngineCreation', () => {
+  const DEFAULT_VALUES = {
+    name: '',
+    rawName: '',
+    language: 'Universal',
+  };
+
+  describe('default values', () => {
+    beforeEach(() => {
+      jest.clearAllMocks();
+    });
+
+    it('renders', () => {
+      setMockValues(DEFAULT_VALUES);
+      const wrapper = shallow(<EngineCreation />);
+      expect(wrapper.find('[data-test-subj="EngineCreation"]')).toHaveLength(1);
+    });
+
+    it('contains a form', () => {
+      setMockValues(DEFAULT_VALUES);
+      const wrapper = shallow(<EngineCreation />);
+      expect(wrapper.find('[data-test-subj="EngineCreationForm"]')).toHaveLength(1);
+    });
+
+    it('contains a name input', () => {
+      setMockValues(DEFAULT_VALUES);
+      const wrapper = shallow(<EngineCreation />);
+      expect(wrapper.find('[data-test-subj="EngineCreationNameInput"]')).toHaveLength(1);
+    });
+
+    it('contains a language input', () => {
+      setMockValues(DEFAULT_VALUES);
+      const wrapper = shallow(<EngineCreation />);
+      expect(wrapper.find('[data-test-subj="EngineCreationLanguageInput"]')).toHaveLength(1);
```

Comment:
I strongly think these should all just be in the default `it('renders')` test. They're not doing anything differently per-test (changing values, props, etc.), so it's not really adding significant value to have a bunch of different tests:

```suggestion
      const wrapper = shallow(<EngineCreation />);

      expect(wrapper.find('[data-test-subj="EngineCreation"]')).toHaveLength(1);
      expect(wrapper.find('[data-test-subj="EngineCreationForm"]')).toHaveLength(1);
      expect(wrapper.find('[data-test-subj="EngineCreationNameInput"]')).toHaveLength(1);
      expect(wrapper.find('[data-test-subj="EngineCreationLanguageInput"]')).toHaveLength(1);
```

If you very strongly feel that the test names add extra context & value (which I can see, but am a little 50/50 on), you could do something like this:

```js
describe('renders', () => {
  let wrapper: ShallowWrapper;

  beforeAll(() => {
    setMockValues(DEFAULT_VALUES);
    wrapper = shallow(<EngineCreation />);
  });

  it('contains a name input', () => {
     expect(wrapper.find('[data-test-subj="EngineCreationNameInput"]')).toHaveLength(1); 
  });

  // ... etc.
});
```

Which makes each it block a little quicker to parse